playground/static/main.js

```diff
@@ -265,22 +271,43 @@ async function updateOutput(input) {
                 errors.forEach(addError);
                 warnings.forEach(addWarning);
 
+                overleafButton.style.display = "none";
+
                 debugEditor.container.style.display = "block";
                 renderIframe.style.display = "none";
                 render.style.display = "none";
 
+                overleafButton.style.display = "none";
+
                 debugEditor.session.setMode("");
                 debugEditor.setValue(content);
                 debugEditor.getSession().selection.clearSelection();
             }
                 break;
+            case "latex":
+                let { content, warnings, errors } = JSON.parse(await compilerAction({ type: "transpile", source: input, format: "latex" }));
+                errors.forEach(addError);
+                warnings.forEach(addWarning);
+
+                overleafButton.style.display = "inline-block";
+
```

Comment:
Maybe remove this newline to improve style, so that all "set style"-statements stick together